src/components/TasksList/TasksList.tsx

```diff
@@ -1,9 +1,35 @@
 import React from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Tabs from '@material-ui/core/Tabs';
+import Tab from '@material-ui/core/Tab';
+import TabPanel from './TabPanel';
 
 const TasksList: React.FC = () => {
+  const [value, setValue] = React.useState(0);
+
+  const handleChange = (event: React.ChangeEvent<{}>, newValue: number) => {
+    setValue(newValue);
+  };
+
   return (
     <div data-testid='task-list-page'>
-      <h1>Tasks List</h1>
+      <AppBar component='section' position='static' color='default'>
+        <Tabs
+          value={value}
+          onChange={handleChange}
+          indicatorColor='primary'
+          textColor='primary'
+          variant='fullWidth'>
+          <Tab label='Newbie Tasks' />
+          <Tab label='My tasks' />
```

Comment:
I would like to have this as an array const:
`const Tabs = [ 
  { label: 'Newbie Tasks', component: 'Newbie Tasks Component' }, 
  { label: 'My tasks', component: 'My Tasks Component' } 
];`

And use it in render like this: 
`{Tabs.map(({ label }, key) => (
   <Tab key={key}>{label}</Tab>
))}`